packages/flutter_web_plugins/lib/src/navigation/url_strategy.dart

```diff
@@ -63,9 +63,15 @@ class PathUrlStrategy extends ui_web.HashUrlStrategy {
 
   @override
   String prepareExternalUrl(String internalUrl) {
-    if (internalUrl.isNotEmpty && !internalUrl.startsWith('/')) {
-      internalUrl = '/$internalUrl';
+    if (internalUrl.isEmpty) {
+      internalUrl = '/';
     }
+    assert(
+      internalUrl.startsWith('/'),
+      "When using PathUrlStrategy, all route names must start with '/' because "
+      "the browser's pathname always starts with '/'. "
```

Comment:
```suggestion
      "the browser's path name always starts with '/'. "
```

Reply:
I intentionally used "pathname" because that's how it's referred to in documentation: https://developer.mozilla.org/en-US/docs/Web/API/URL/pathname